tensorboard/uploader/uploader_main.py

```diff
@@ -367,9 +367,15 @@ def execute(self, channel):
       msg = 'Output directory already exists: %r' % outdir
       raise base_plugin.FlagsError(msg)
     num_experiments = 0
-    for experiment_id in exporter.export():
-      num_experiments += 1
-      print('Downloaded experiment %s' % experiment_id)
+    try:
+      for experiment_id in exporter.export():
+        num_experiments += 1
+        print('Downloaded experiment %s' % experiment_id)
+    except exporter_lib.NewExceptionType as e:
```

Comment:
`NewExceptionType` should be `GrpcTimeoutException`. Can you try
manually running an end-to-end test to make sure that the main module is
wired up correctly?

Reply:
Done.